components/server/src/user/usage-service.ts

```diff
@@ -28,4 +32,11 @@ export class UsageService {
             usageLimit: costCenter?.spendingLimit || 0,
         };
     }
+
+    async getCurrentBillingStategy(attributionId: AttributionId): Promise<CostCenter_BillingStrategy | undefined> {
```

Comment:
A more general version (which allows for more composition) is to return the CostCenter and let the caller compare/check the billing strategy themselves.